Examples/Paths/Array-element-paths.fsh

```diff
@@ -0,0 +1,4 @@
+@Name: Array Element Paths
```

Comment:
Doh.  I totally missed this when I reviewed, but technically the above statement should be in a comment.  This happens in several of the other files too.

@markkramerus -- can you submit another PR fixing this?